x-pack/test/case_api_integration/security_and_spaces/tests/trial/configure/get_connectors.ts

```diff
@@ -38,7 +40,7 @@ export default ({ getService }: FtrProviderContext): void => {
         .set('kbn-xsrf', 'true')
         .send({
           name: 'An email action',
-          actionTypeId: '.email',
+          connector_type_id: '.email',
```

Comment:
Actions api changed a couple months ago from `actionTypeId` to `connector_type_id`.